### ISSUE TITLE
How to hide the huge title inside the note?

### ISSUE THREAD

**Issue by Flexit07** [Make-md/makemd#69]
After the last release, "Make" forces me to have the title of the note with giants font on the upper side of the note despite of I switched off the option

There's a way to hide it? I can't find anything

Thanks

**Comment by jp-cen** [comment]
That's the inline context feature which you can turn off. An update will address making it respect the Obsidian setting

**Comment by Flexit07** [comment]
Thanks for your fast reply but it doesn't seem to work.
And if I switch on the corresponding option in the default setting... I see it twice as big.
I tried with another vault with completely different settings and plugins, but the result is the same
![Clipboardfgfgff](https://user-images.githubusercontent.com/126170116/221374792-a0c11b52-a333-4761-a93e-95d0a44e3a55.jpg)

**Comment by jp-cen** [comment]
Did you try restarting Obsidian? Right now settings don’t auto-apply without restarting.

**Comment by jp-cen** [comment]
The inline title is a setting in Obsidian Appearances, check to see if that’s turned on

**Comment by Flexit07** [comment]
As I wrote, this is what happens if i turn on the setting in Obsidian Appearances too

---

![2023-02-25_200236](https://user-images.githubusercontent.com/126170116/221375068-462dfa85-8454-41dd-8bae-a538c0e78322.jpg)

**Comment by jp-cen** [comment]
I see, are you in reading mode? There are some issues with inline title not showing properly and not respecting settings, will be pushing a fix today to fix it.

**Comment by Flexit07** [comment]
Yes, I see it double only in reading mode, thanks for your kindness and compliments for the plugin, you are a genius

**Comment by MarcosDantasdeSantana** [comment]
I have a simillar problem of disabled configuration not working i have opened a issue.

**Comment by RAFA3L** [comment]
This is a wonderful plugin and I was playing with the settings today, would be nice an option to hide the metatag in all the view modes, to use it only in the sidebar

![2023-02-26_14-15-22](https://user-images.githubusercontent.com/3952265/221429058-3a1fb215-f723-421c-9bc8-73a6d0528241.jpg)
![2023-02-26_14-13-17](https://user-images.githubusercontent.com/3952265/221429064-3fb29743-12a1-40db-b76e-5bfafdd0a32c.jpg)
![2023-02-26_14-09-21](https://user-images.githubusercontent.com/3952265/221429066-8a97e8f1-70a6-425c-9a4f-587fb645af54.jpg)

**Comment by rebron1900** [comment]
> This is a wonderful plugin and I was playing with the settings today, would be nice an option to hide the metatag in all the view modes, to use it only in the sidebar这是一个很棒的插件，我今天正在玩这些设置，这是一个很好的选项，可以在所有视图模式下隐藏元标记，只在侧边栏中使用它
> 
> ![2023-02-26_14-15-22](https://user-images.githubusercontent.com/3952265/221429058-3a1fb215-f723-421c-9bc8-73a6d0528241.jpg) ![2023-02-26_14-13-17](https://user-images.githubusercontent.com/3952265/221429064-3fb29743-12a1-40db-b76e-5bfafdd0a32c.jpg) ![2023-02-26_14-09-21](https://user-images.githubusercontent.com/3952265/221429066-8a97e8f1-70a6-425c-9a4f-587fb645af54.jpg)

How do you get the YAML region to display the region to the right?

**Comment by RAFA3L** [comment]
This is my config in context
![2023-02-28_21-43-01](https://user-images.githubusercontent.com/3952265/222023185-578a0f00-c2dc-425e-9b7e-bd80a8126ce7.png)

**Comment by rebron1900** [comment]
![企业微信截图_16776353706255](https://user-images.githubusercontent.com/7929884/222023976-023f6499-dc66-46f2-823c-4ade525ad118.png)

After I set it is

**Comment by RAFA3L** [comment]
Maybe it is necessary to open the panel from the command "Open Context Explorer" the first time

**Comment by rebron1900** [comment]
> Maybe it is necessary to open the panel from the command "Open Context Explorer" the first time

thks，It work
